### PR TITLE
kirkstone: initramfs-test-image: add adbd support

### PR DESCRIPTION
**Pull request by eberman-quic**
This is a rebased copy of #397 to kirkstone branch

---
 
Install and enable adbd if meta-openmebedded is enabled.

### CODE REVIEW

File: conf/machine/include/qcom-common.inc [hunk 1]
```diff
@@ -24,6 +24,8 @@ PREFERRED_PROVIDER_virtual/kernel ??= "linux-linaro-qcomlt"
 
 PREFERRED_PROVIDER_android-tools-conf = "android-tools-conf-configfs"
 
+PREFERRED_PROVIDER_android-tools-conf = "android-tools-conf-configfs"
+
```
Comment (lumag) [inline conf/machine/include/qcom-common.inc:28]: The same line is present two lines above. Are you sure that you want to repeat it?

File: dynamic-layers/openembedded-layer/recipes-devtools/android-tools/android-tools-adbd-cmdline.bb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+DESCRIPTION = "An override for adbd unit - start adbd depending on the kernel command line"
```
Comment (lumag) [inline dynamic-layers/openembedded-layer/recipes-devtools/android-tools/android-tools-adbd-cmdline.bb:1]: This is not a part of #227, so PR cover letter is wrong.

**Review by lumag — CHANGES_REQUESTED** [review]
Aside of the other comments, the commits miss your SoB trailers.

File: dynamic-layers/openembedded-layer/recipes-devtools/android-tools/android-tools-adbd-cmdline.bb (hunk 2 above)
Comment (eberman-quic) [reply dynamic-layers/openembedded-layer/recipes-devtools/android-tools/android-tools-adbd-cmdline.bb:1]: Apologies, found the wrong PR in search. Should be fixed properly now.

**Review by lumag — APPROVED** [review]

**Pull request merged**